packages/web-components/tsconfig.bindings.json

```diff
@@ -16,7 +16,14 @@
     "sourceMap": true,
     "jsx": "react",
     "target": "es5",
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "typeRoots": ["../../node_modules/@types"],
+    "paths": {
+      "react": ["../../node_modules/@types/react"],
+      "react-dom": ["../../node_modules/@types/react-dom"],
+      "@types/react": ["../../node_modules/@types/react"],
+      "@types/react-dom": ["../../node_modules/@types/react-dom"]
```

Comment:
These are needed to enforce the react version from this package, rather than using another version from the workspace